packages/account/src/Sections/Security/Passkeys/passkeys.scss

```diff
@@ -15,16 +15,22 @@
             align-items: center;
             margin-block: 1.6rem;
 
+            .dc-input--error {
+                margin-bottom: 3.2rem;
+            }
+
             &--expanded {
                 display: block;
 
-                .dc-icon {
-                    display: block;
-                    margin-inline: auto;
-                }
+                .passkey-status {
+                    &__icon {
+                        display: block;
+                        margin-inline: auto;
+                    }
 
-                .passkeys-status__title {
-                    margin-bottom: 0;
+                    &__title {
+                        margin-bottom: 0;
+                    }
                 }
             }
         }
```

Comment:
Edit margin and padding between line 36 - line 121 🙏

Reply:
@suisin-deriv , may I please ask for your assistance when you are available 🙏

Reply:
@yauheni-deriv basically we need to use logical properties so that the UI works fine in RTL layouts as well.
you can check it here: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values

<img width="512" alt="Screenshot 2024-05-24 at 6 12 47 PM" src="https://github.com/binary-com/deriv-app/assets/84661147/8fb72af0-4b07-4eec-bb52-fd839ffbab7f">

Reply:
@suisin-deriv , @amina-deriv , added fixes for styling in second PR:
https://github.com/yauheni-deriv/deriv-app/pull/44